client/my-sites/plugins/plugin-navbar-footer/index.tsx

```diff
@@ -0,0 +1,389 @@
+										{ translate( 'WordPress Plugins' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/google/">{ translate( 'Google Apps' ) }</a>
+								</li>
+							</ul>
+						</div>
+						<div>
+							<h3>{ translate( 'Resources' ) }</h3>
+							<ul>
+								<li>
+									<a href="https://wordpress.com/support/">
+										{ translate( 'WordPress.com Support' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/forums/">{ translate( 'WordPress Forums' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/blog/">{ translate( 'WordPress News' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/go/">
+										{ translate( 'Website Building&nbsp;Tips' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/business-name-generator/">
+										{ translate( 'Business Name Generator' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/logo-maker/">{ translate( 'Logo Maker' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/webinars/">{ translate( 'Daily Webinars' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/courses/">{ translate( 'WordPress Courses' ) }</a>
+								</li>
+								<li>
+									<a href="https://developer.wordpress.com/" data-is_external="1">
+										{ translate( 'Developer' ) }{ ' ' }
+										<span className="lp-link-chevron-external">{ translate( 'Resources' ) }</span>
+									</a>
+								</li>
+							</ul>
+						</div>
+						<div>
+							<h3>{ translate( 'Company' ) }</h3>
+							<ul>
+								<li>
+									<a href="https://wordpress.com/about/">{ translate( 'About' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/partners/">{ translate( 'Partners' ) }</a>
+								</li>
+								<li>
+									<a href="https://automattic.com/press/" data-is_external="1">
+										<span className="lp-link-chevron-external">{ translate( 'Press' ) }</span>
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/tos/">{ translate( 'Terms of Service' ) }</a>
+								</li>
+								<li>
+									<a href="https://automattic.com/privacy/" data-is_external="1">
+										{ translate( 'Privacy' ) }{ ' ' }
+										<span className="lp-link-chevron-external">{ translate( 'Policy' ) }</span>
+									</a>
+								</li>
+							</ul>
+						</div>
+					</div>
+					<div className="lpc-footer-subnav-container">
+						<div className="lp-footer-language">
+							<h2 className="lp-hidden">{ translate( 'Language' ) }</h2>
+							<div className="lp-language-picker">
+								<div className="lp-language-picker__icon"></div>
+								<div className="lp-language-picker__chevron"></div>
+								<select
+									className="lp-language-picker__content"
+									title={ translate( 'Change Language' ) }
```

Comment:
To match the LOHP, I think we need to default to the current language instead of `Change Language`.
This `select` also doesn't seem to work and the links still point to LOHP.
```suggestion
									title={ translate( 'Change Language' ) }
									onChange={ ( e ) => navigate( e.target.value ) }
```